crates/pop-contracts/src/utils/helpers.rs

```diff
@@ -1,33 +1,40 @@
-use anyhow::{anyhow, Result};
 use contract_build::ManifestPath;
 use contract_extrinsics::BalanceVariant;
 use ink_env::{DefaultEnvironment, Environment};
 use std::{path::PathBuf, str::FromStr};
 use subxt::{Config, PolkadotConfig as DefaultConfig};
+use thiserror::Error;
 
-// If the user specifies a path (which is not the current directory), it will have to manually
-// add a Cargo.toml file. If not provided, pop-cli will ask the user for a specific path. or ask
-// to the user the specific path (Like cargo-contract does)
-pub fn get_manifest_path(path: &Option<PathBuf>) -> anyhow::Result<ManifestPath> {
-	if path.is_some() {
-		let full_path: PathBuf =
-			PathBuf::from(path.as_ref().unwrap().to_string_lossy().to_string() + "/Cargo.toml");
+#[derive(Error, Debug)]
+pub enum Error {
```

Comment:
Why do individual modules have their own definition of an `Error` enum? For one crate, the idiomatic design is to have a single `error.rs` file, and collate all errors into that one enum. You can use struct variants if you feel one variant of `Error` can have multiple applications but fall under the same module.

Reply:
Done.